animations/explicit_widgets.md

```diff
@@ -0,0 +1,166 @@
+7. Start the animation.  
+8. Define the <code>BuildContext</code> to return the animation.  
+9. Dispose and stop the animation.   -->
+
+
+<div>
+<table class="table" width="100%">
+  <col width="35%">
+  <col width="65%">
+	<tbody>
+    <tr>
+      <td><img src="images/explicit.png" alt="Explicit animation widgets"></td>
+      <td>
+      1. Declare your <code>StatefulWidget</code> class.<br>
+      2. Declare the <code>State</code> class with the appropriate ticker: <code>SingleTickerProviderStateMixin</code> or <code>TickerProviderStateMixin</code>.<br>
+      3. Declare an instance of <code>Animation&lt;T&gt;</code> and <code>AnimationController</code>.<br>
+      4. Define optional and required parameters such as <code>vsync</code> and <code>duration</code>.<br>
+      5. Specify the animation such as <code>CurvedAnimation</code>. <br>
+      6. Add the appropriate action in the <code>listener</code> callback such as <code>setState</code>. <br>
+      7. Start the animation.  <br>
+      8. Define the <code>BuildContext</code> to return the animation. <br>
+      9. Dispose and stop the animation. <br>
+      </td>
+    </tr>
+   </tbody>
+  </table>
+</div>
+
+<!-- To reuse existing Flutter SDK implicit and transition widgets, see [Implicit animations](/animations/implicit_widgets.html) and [Transition animations](/animations/transition_widgets.html). -->
+
+## Explicit animation examples
+
+[Tutorial: Animations in Flutter](/tutorials/animation/)— < revise the tutorial and transfer it to an example here -
+This example shows you how to build an explicit animation using Flutter concepts, classes, and methods—and it walks you through 5 animation examples that build on each other to make a complex animation. The example also shows how to use the `AnimationWidget` and the `AnimatedBuilder`.
+> This tutorial needs to be revised to remove the concept descriptions that are described on the Concepts web page to keep definitions together—so that information isn't repeated or contradicted on multiple web pages. For example, the AnimationController and Tween are defined on three separate web pages.
+> The objective of the tutorial s/b revised to state: 1. How to use Flutter animation classes to add an explicit animation to a widget 2. When to use the `AnimatedWidget` and when to use the `AnimatedBuilder`. (perhaps make the second part a separate tutorial?)
+
+
+The code for the `animationexample` example is shown below. The `animationexample` widget extends the `AnimatedWidget` class and already includes the listeners, tickers, and other elements that define the animation.  
+<!-- skip -->
+{% prettify dart %}
+```Dart
+import 'package:flutter/material.dart';
+
+add the code for the explicit widget
+
+```
+{% endprettify %}
+
+
+
+## Explicit animations widgets
+
+The Flutter SDK includes the following explicit animation widgets. This is a sample of the many explicit widgets that are included in the Flutter SDK.
```

Comment:
The widgets in the list below are widgets _using_ explicit animations in their implementations, not widgets for _building_ explicit animations. Using the widgets in the list require no knowledge about building animations. Since this doc is about creating explicit animations using AnimatedBuilder, I don't feel this list is relevant, unless the intent is to point the user to the source code of these widgets as references for writing explicit animations.